env/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@okta/env",
+  "version": "1.0.0",
+  "main": "index.js",
+  "private": true
+}
```

Comment:
should add devDependency for "dotenv"

Reply:
actually should be regular dependency. Then the eslint for extraneous require will not be needed.